cmd/eval.go

```diff
@@ -54,6 +54,7 @@ func validateFile(file io.Reader) error {
 	}
 
 	var holder interface{}
+
 	return yaml.Unmarshal(fileBytes, &holder)
 }
```

Comment:
I would say that the `validateFile` function should no longer be necessary. We should instead be validating within the `template` package before this is needed. Basically, when we call the `templateBuilder.Build()` I'd say we should just go ahead and call the template's validate command which would cover this. Thoughts?

Reply:
I am perfectly fine with removing the validation function from `eval.go`. From what I can tell, in the previous iteration of validation, we also were validating the output of `combinedTemplate` which is why the function was left over.

Reply:
I think we can honestly get rid of the `combinedTemplate` and the validation function as we are essentially adding explicitly valid YAML to whatever is inside of the template.